quantum-template.tex

```diff
@@ -171,6 +169,11 @@ \section{Unpublished and accepted works}
 It can be used for works not yet accepted in Quantum or not even intended for submission to Quantum.
 On the contrary, by giving the \texttt{accepted=YYYY-MM-DD} option, with \texttt{YYYY-MM-DD} the acceptance date, the note ``Accepted in Quantum YYYY-MM-DD, click title to verify'' can be added to the bottom of each page to clearly mark works that have been accepted in Quantum. 
 
+\section{Plots}\label{sec:plots}
+Quantum provides a Jupyter notebook based on the widely used library \texttt{matplotlib} that greatly simplifies the creation of plots that integrate seamlessly with the \texttt{quantumarticle} class. This is intended as a service to the authors and is \textit{not} mandatory. The notebook can be found in the \href{https://github.com/quantum-journal/quantum-journal}{GitHub repository}. The Python notebook relies. It only requires the author to specify font size and paper format that were passed as options to \texttt{quantumarticle}.
```

Comment:
```suggestion
Quantum provides a \href{https://jupyter.org/}{Jupyter notebook} based on the widely used \href{https://matplotlib.org/}{matplotlib} library that greatly simplifies the creation of plots that integrate seamlessly with the \texttt{quantumarticle} class. This is intended as a service to the authors, is \textit{not} mandatory, and currently in beta stage. You can \href{https://github.com/quantum-journal/quantum-journal/blob/quantum-plots/quantum-plots.ipynb}{preview} and download the \href{https://raw.githubusercontent.com/quantum-journal/quantum-journal/master/quantum-plots.ipynb}{quantum-plots.ipynb} notebook and accompanying \href{https://raw.githubusercontent.com/quantum-journal/quantum-journal/master/quantum-plots.mplstyle}{quantum-plots.mplstyle} file from the \href{https://github.com/quantum-journal/quantum-journal}{quantumarticle GitHub repository}. You only need to specify the font size and paper format that were passed as options to \texttt{quantumarticle} to get plots with fitting font sizes.
```

Reply:
How about that? The links should start working once we merge this into master. We should remember to check that this is actually the case...

Reply:
Ah, and maybe can you please and `(beta)` to the title of the Notebook? Thanks!